headapps/MvpSite/MvpSite.Rendering/Views/Shared/_Layout.cshtml

```diff
@@ -68,6 +68,6 @@
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/2.0.2/anime.min.js"></script>
     <script src="https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/js/accordion.js"></script>
-    <script src="/js/mvp-site.es5.min.js?v=1.14"></script>
+    <script src="/js/mvp-site.js?v=1.15"></script>
```

Comment:
You've updated the version number here correctly, but can you please revert it to the .es5.min version again?